src/Handlers/PackageReleases.php

```diff
@@ -61,7 +61,6 @@ public function __invoke(ServerRequestInterface $req): ResponseInterface
             JOIN gossamer_provider_publickeys k ON r.publickey = k.id
             WHERE p.provider = ? AND r.package = ?   
                   AND NOT r.revoked
-                  AND NOT k.revoked
```

Comment:
If someone revokes a key, that doesn't immediately invalidate all old releases. It just means the key cannot be used anymore.

If someone wishes to revoke their old updates too, they MUST do so explicitly.